Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -489,18 +515,35 @@ class DropDownMenu
     }
   }
 
-  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement): void
+  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement, isPeriodic: boolean): void
   {
     var space_top = Math.round (LayoutUtility.GetOffset (referenceElement).top - window.pageYOffset);
     var space_bottom = Math.round (document.documentElement.clientHeight - LayoutUtility.GetOffset (referenceElement).top - LayoutUtility.GetHeight (referenceElement) + window.pageYOffset);
     var space_left = LayoutUtility.GetOffset (referenceElement).left;
     var space_right = document.documentElement.clientWidth - LayoutUtility.GetOffset (referenceElement).left - LayoutUtility.GetWidth (referenceElement);
 
+    // Periodic update: only update our position if it has change to allow the usage of the scrollbars
+    // Other update: it might take a periodic call to get the correct position so we force it by deleting the last position information
```

Comment:
Might be better off in a JSDoc comment to get IDE support

Reply:
Its a private method and this information is only meant to explain the reasoning behind the implementation. But it might be useful to explain the periodic parameter in a JSDoc comment

Reply:
It is private, its still useful to see param docs tho when editing a 1.3k lines file 😅

Reply:
Its just 990 loc 😌